metrics/utils/json_to_athena_schema.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python
```

Comment:
tl;dr: I am strongly for renaming this file to simply ``json_to_schema.py``.

The naming question issue is -- this is a metric specific utility, so right directory IMO. However it is very Athena specific. Since we know we'll have to bridge the service divide for a while, do we want to push down a level and have directories AWS & GCP, with a more generic name 'json_to_schema'. That would let us just flip import statement to migrate.

Yeah, that could be YAGNI, but (a) we already crossed that bridge in ``cfs.py``, and (b) I'd prefer to rename the file now, so we don't need a flag day later.

Reply:
I'll remove this script from the PR so that I can merge it and actually use the transform code in jenkins. We can then discuss what schema utils we should write / merge etc at a later date

Reply:
moved discussion to #5.